packages/teams-js/src/public/runtime.ts

```diff
@@ -179,7 +242,7 @@ export function isRuntimeInitialized(runtime: IBaseRuntime): runtime is Runtime
 export let runtime: Runtime | UninitializedRuntime = _uninitializedRuntime;
 
 export const teamsRuntimeConfig: Runtime = {
-  apiVersion: 2,
+  apiVersion: 3,
```

Comment:
I suspect that not using latest here is best because it forces the person making the update to intentionally decide how to handle the Teams config, and it's not clear to me that "how to handle it" is going to be the same every time. Having someone look and make a decision feels safest.